src/cache/cache_hs.c

```diff
@@ -751,30 +755,32 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
             /* We only need to clear the flag on the updates up to where the error occurs. */
             err_pos = WT_MIN(multi->supd_entries - 1, supd_index);
 
-            /* Traverse the keys again to clear the flags on updates inserted to lookaside. */
+            /* Traverse the keys again to clear the flags on updates inserted to the history store.
+             */
             for (supd_index = 0, list = multi->supd; supd_index <= err_pos; ++supd_index, ++list) {
                 if (list->onpage_upd == NULL)
                     continue;
 
                 /*
                  * Clear the flag on all updates in the chain regardless of whether we wrote the
-                 * entry to lookaside. That means we might clear the flag for updates that are in
-                 * lookaside, but that's safe and getting this correct is not worth the complexity
-                 * of tracking which specific updates were written in the transaction.
+                 * entry to the history store. That means we might clear the flag for updates that
+                 * are in the history store, but that's safe and getting this correct is not worth
+                 * the complexity of tracking which specific updates were written in the
+                 * transaction.
```

Comment:
transaction can be combined with the previous line.

Reply:
Nah the full stop goes over.